topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -455,23 +506,18 @@ the two samples?
 {: .hands_on}
 
 
-To further explore the community structure, we can visualize it with dedicated tools such as Phinch:
+To further explore the community structure, we can visualize it with dedicated tools such as Phinch.
 
 > ### {% icon hands_on %} Hands-on: Visualization of the community structure with Phinch
 >
 > 1. **Make.biom** {% icon tool %} with the following parameters
->   - "shared" to Make.shared
->   - "constaxonomy" to taxonomy output from the first run of `classify.otu` (collection)
+>   - "shared" to `Make.shared` output
+>   - "constaxonomy" to taxonomy output from the first run of `Classify.otu` (collection)
+> 1. On "calculator" collection, click on "Visualize" and then "Phinch"
+> 2. Play with the several interactive visualisations:
 >
->    The Galaxy project runs an instance of Phinch, and if you look at the output BIOM file, you will see a link to view the file at Phinch:
+> ![Phinch website interface](../../../../shared/images/phinch_overviewpage.png "Phinch visualizations")
```

Comment:
perhaps we should add a little note box here to explain what to do if that link is not present on the server (since it is not turned on by default) ..i.e. alternative is to download the biom file and upload directly to phinch server (http://phinch.org)